sage-ios/SAGE/SAGE/EditProfileController.swift

```diff
@@ -0,0 +1,136 @@
+        editProfileView.setProfileImage(self.user)
+        
+        editProfileView.school.button.addTarget(self, action: "schoolButtonTapped", forControlEvents: .TouchUpInside)
+        editProfileView.hours.button.addTarget(self, action: "hoursButtonTapped", forControlEvents: .TouchUpInside)
+        editProfileView.photoButton.addTarget(self, action: "chooseNewPhoto", forControlEvents: .TouchUpInside)
+    }
+    
+    @objc private func chooseNewPhoto() {
+        let actionSheet = UIAlertController(title: nil, message: nil, preferredStyle: .ActionSheet)
+        let imagePickerController: UIImagePickerController = UIImagePickerController()
+        imagePickerController.delegate = self
+        imagePickerController.allowsEditing = false
+        
+        actionSheet.addAction(UIAlertAction(title: "Take Photo", style: .Default, handler: { (alertAction) -> Void in
+            if UIImagePickerController.isSourceTypeAvailable(.Camera) {
+                imagePickerController.sourceType = .Camera
+                self.presentViewController(imagePickerController, animated: true, completion: nil)
+            }
+        }))
+        
+        actionSheet.addAction(UIAlertAction(title: "Choose From Library", style: .Default, handler: { (alertAction) -> Void in
+            if UIImagePickerController.isSourceTypeAvailable(.PhotoLibrary) {
+                imagePickerController.sourceType = .PhotoLibrary
+                self.presentViewController(imagePickerController, animated: true, completion: nil)
+            }
+        }))
+        actionSheet.addAction(UIAlertAction(title: "Cancel", style: .Cancel, handler: { (alertAction) -> Void in
+            actionSheet.dismissViewControllerAnimated(true, completion: nil)
+        }))
+        
+        self.presentViewController(actionSheet, animated: true, completion: nil)
+    }
+    
+    @objc private func schoolButtonTapped() {
+        let tableViewController = SelectSchoolEditProfileController()
+        tableViewController.parentVCEditProfile = self
+        if let topItem = self.navigationController?.navigationBar.topItem {
+            topItem.backBarButtonItem = UIBarButtonItem(title: "", style: .Plain, target: nil, action: nil)
+        }
+        self.navigationController?.pushViewController(tableViewController, animated: true)
+    }
+    
+    @objc private func hoursButtonTapped() {
+        let tableViewController = SelectHoursEditProfileController()
+        tableViewController.parentVCEditProfile = self
+        if let topItem = self.navigationController?.navigationBar.topItem {
+            topItem.backBarButtonItem = UIBarButtonItem(title: "", style: .Plain, target: nil, action: nil)
+        }
+        self.navigationController?.pushViewController(tableViewController, animated: true)
+    }
+    
+    func didSelectSchool(school: School) {
+        self.user.school = school
+        (self.view as? EditProfileView)?.displaySchoolName(school.name!)
```

Comment:
if you have an editProfileView property this ugly line and similar ones in this file will also not be ugly